index.js

```diff
@@ -113,6 +109,7 @@ const staticify = (root, options) => {
     };
 
     const serve = req => {
+        // eslint-disable-next-line node/no-deprecated-api
         const filePath = stripVersion(url.parse(req.url).pathname);
```

Comment:
I couldn't make this work with relative URLs so for now I just disabled the error. We should definitely revisit, or if anyone has a working solution let me know.

Reply:
I think the `url.parse(req.url).pathname` needs to change to `(new URL(req.url)).pathname`. Worth trying it to be sure.

Reply:
That returns `undefined`, I already tried it.

Reply:
We need to pass the base option for this to work...